packages/manager/modules/pci/src/projects/project/storages/databases/database/service-integration/service-integration.html

```diff
@@ -0,0 +1,92 @@
+    <oui-message
+        data-ng-if="$ctrl.addableServicesList.length === 0"
+        data-type="warning"
+        data-dismissable
+    >
+        <span
+            data-translate="pci_databases_service_integration_tab_no_kafka_service"
+        ></span>
+    </oui-message>
+
+    <p data-translate="pci_databases_service_integration_tab_descritpion"></p>
+    <oui-datagrid
+        class="mt-5"
+        data-page-size="10"
+        data-rows="$ctrl.serviceIntegrationList"
+    >
+        <oui-datagrid-column
+            data-title=":: 'pci_databases_service_integration_tab_kafka_service' | translate"
+            data-property="serviceName"
+            data-sortable="asc"
+            data-type="string"
+            data-searchable
+            data-filterable
+        ></oui-datagrid-column>
+        <oui-datagrid-column
+            data-title=":: 'pci_database_common_status' | translate"
+            data-property="status"
+        >
+            <span
+                class="oui-badge"
+                data-ng-class="{
+                    'oui-badge_success': ($row.isStatusGroupReady()),
+                    'oui-badge_warning': ($row.isStatusGroupPending()),
+                    'oui-badge_error': ($row.isStatusGroupError()),
+                }"
+            >
+                <span
+                    data-ng-bind=":: ('pci_database_common_status_' + $row.status) | translate"
+                ></span>
+            </span>
+        </oui-datagrid-column>
+        <oui-datagrid-column class="content-r">
+            <div class="text-right">
+                <button
+                    data-ng-disabled="$row.statusGroup !== 'READY'"
+                    class="oui-button oui-button_s oui-button_secondary"
+                    data-ng-click="$ctrl.trackAndDeleteServiceIntegration($row)"
+                >
+                    <span
+                        class="oui-icon oui-icon-bin"
+                        aria-hidden="true"
+                    ></span>
+                </button>
+            </div>
+        </oui-datagrid-column>
+        <oui-datagrid-topbar>
+            <oui-button
+                data-disabled="$ctrl.addableServicesList.length === 0"
+                data-on-click="$ctrl.trackAndAddServiceIntegration()"
+            >
+                <span
+                    class="oui-icon oui-icon-add pr-1"
+                    aria-hidden="true"
+                ></span>
+                <span
+                    data-translate="pci_databases_service_integration_tab_add_kafka"
+                ></span>
+            </oui-button>
+            <button
+                data-ng-if="$ctrl.serviceIntegrationList.length > 0"
+                class="float-right oui-button oui-button_secondary oui-button_icon-only oui-button_s"
+                type="button"
+                data-title="{{:: 'pci_databases_common_refresh' | translate }}"
+                data-ng-click="$ctrl.refreshing=true; $ctrl.refreshServiceIntegration()"
```

Comment:
More than one action have to be implemented into a dedicated function.

```suggestion
                data-ng-click="$ctrl.onRefreshServiceIntegrationClick()"
```

Reply:
Changed here: https://github.com/ovh/manager/pull/5765/commits/d401f3622207c4c8282bc3619d16efe2d2210000